### ISSUE TITLE
Commons compress CVE

### ISSUE THREAD

**Issue by KlemenDEV** [jcefmaven/jcefmaven#111]
This library uses apache commons compress library that should be updated because the current one used is using vulnerable version:

![image](https://github.com/user-attachments/assets/e46c0d36-57b8-42bd-82b1-1942499e13f3)

(https://mvnrepository.com/artifact/me.friwi/jcefmaven/127.3.1)

**Comment by FriwiDev** [comment]
I bumped the dependencies now, even though this library is not really affected by these CVEs (I have viewed them previously, as there were also github alerts on this). One is a resource allocation and one is concerning an infinite loop. As we know which files we are downloading specifically, triggering it would require someone to inject wrong files into the download, which is unlikely. But even if they were to succeed, this vulnerability would mainly be critical for server applications (DoS), which will in turn most likely not use this library. So overall, these CVEs do not really affect us in any meaningful way.

I will start a new build with the bumped libraries when the next jcef update happens.
